server/bad_words.txt

```diff
@@ -0,0 +1,149 @@
+ass
```

Comment:
You might want to alphabetize these so it's easy to add more in the future
In a terminal you can use the `sort` command line tool